cli/lib/kontena/cli/nodes/update_command.rb

```diff
@@ -3,18 +3,42 @@ class UpdateCommand < Kontena::Command
     include Kontena::Cli::Common
     include Kontena::Cli::GridOptions
 
-    parameter "NODE_ID", "Node id"
+    requires_current_master
+    requires_current_master_token
+    requires_current_grid
+
+    parameter "NAME", "Node name"
+
+    option ["--token"], "TOKEN", "Node token"
+    option "--generate-token", :flag, "Generate new node token"
+
     option ["-l", "--label"], "LABEL", "Node label", multivalued: true
+    option "--clear-labels", :flag, "Clear node labels"
```

Comment:
Are we using `--clear-xyz` somewhere else? Just check that this is consistent with other commands.